qcodes/instrument_drivers/yokogawa/Yokogawa_GS200.py

```diff
@@ -0,0 +1,698 @@
+from functools import partial
```

Comment:
is it worth it copying the whole driver? wouldn't  a subclass with old name work as well (same with the `_float_round`)?

Reply:
Yes it could be done but being very defensive someone could depend on that the name of the class of the measurement module inside the driver so I would need to customize it such that the classes of the module attributes stayed the same. That did not feel worth the risk of getting it wrong

Reply:
@astafan8 Does that tradeoff seem ok to you?

Reply:
(sorry, forgot to respond) yes, it's good!